assets/js/app.js

```diff
@@ -129,7 +135,7 @@ window.setInterval(function() {
 // App & Routing
 //
 
-const main = Vue.component('main', {
+const main = Vue.component('Main', {
```

Comment:
Best practice?

Reply:
Ja, gernell kenn ich das so, dass man Vue Komponenten groß schreibt. Hier gabs konkret das Problem, dass es in HTML 5 ein `main` Element gibt und Vue über den Konflikt meckert.

Reply:
Dann sollten wir das konsequent für alle Komponenten machen?

Reply:
Ja, sollten wir. Wollte ich aber nicht in diesem PR machen und hatte kein schöneres Wort als "main" parat. Ich würde ja sowieso gerne perspektivisch auf Single File Componentes refactorn. Das wäre dann sicher ein guter Zeitpunkt für die Namensänderung.